content/en/blog/_posts/2021-11-02-csi-migration-status.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: blog
+title: "Kubernetes 1.23: Kubernetes In-Tree to CSI Volume Migration Status Update"
+date: 2021-11-03T15:00:00+08:00
+slug: kubernetes-feature-csi-migration-status-update
+---
+
+**Authors:** Jiawei Wang, Software Engineer, Google
+
+The Kubernetes in-tree storage plugin to [Container Storage Interface (CSI)](https://kubernetes.io/blog/2019/01/15/container-storage-interface-ga/) migration infrastructure has already been [beta](https://kubernetes.io/blog/2019/12/09/kubernetes-1-17-feature-csi-migration-beta/) since v1.17. CSI migration was introduced as alpha in Kubernetes v1.14.
```

Comment:
Not entirely sure but I think these links need to be changed to reference style links for consistency.

```suggestion
The Kubernetes in-tree storage plugin to [Container Storage Interface (CSI)](/blog/2019/01/15/container-storage-interface-ga/) migration infrastructure has already been in [beta](/blog/2019/12/09/kubernetes-1-17-feature-csi-migration-beta/) since v1.17. CSI migration was introduced as alpha in Kubernetes v1.14.
```

Reply:
Done